docs/clients/grpc/projections/README.md

```diff
@@ -0,0 +1,665 @@
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+ resetProjection(projectionName: string,options?: ResetProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+Resetting a projection that does not exists will result in an error. 
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Reset_NotFound
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+ resetProjection(projectionName: string,options?: ResetProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Create a projection
+
+Creates a projection that runs until the end of the log and then continues running. The query parameter contains the JavaScript you want created as a projection.
+Projections have explicit names, and you can enable or disable them via this name.
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#CreateContinuous
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+public CompletableFuture createContinuous(final String projectionName, final String query)
+public CompletableFuture createContinuous(final String projectionName, final String query, CreateContinuousProjectionOptions options)
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+createContinuousProjection(projectionName: string, query: string, options?: CreateContinuousProjectionOptions): Promise<void>
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+Trying to create projections with the same name will result in an error:
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#CreateContinuous_Conflict
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+public CompletableFuture createContinuous(final String projectionName, final String query)
+public CompletableFuture createContinuous(final String projectionName, final String query, CreateContinuousProjectionOptions options)
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+createContinuousProjection(projectionName: string, query: string, options?: CreateContinuousProjectionOptions): Promise<void>
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Update a projection
+
+Updates a projection. The name parameter is the name of the projection to be updated. The query parameter contains the new JavaScript.
```

Comment:
**SUGGESTION:** Would be good to add some context to this operation (the same apply to the methods below).